test/WebJobs.Script.Tests/SanitizerTests.cs

```diff
@@ -41,6 +41,9 @@ public class SanitizerTests
         [InlineData("test,aaa://aaa:aaaaaa1111aa@aaa.aaa.io:1111,test", "test,[Hidden Credential],test")]
         [InlineData(@"some text abc://abc:aaaaaa1111aa@aaa.abc.io:1111 some text abc://abc:aaaaaa1111aa@aaa.abc.io:1111 text", @"some text [Hidden Credential] some text [Hidden Credential] text")]
         [InlineData(@"some text abc://abc:aaaaaa1111aa@aaa.abc.io:1111 some text AccountKey=heyyyyyyy text", @"some text [Hidden Credential] some text [Hidden Credential]")]
+        [InlineData("""{"queueName":"my-q-items","connection":"MyConnection","type":"queueTrigger","name":"qTrigger1","direction":"in"}""", "{\"queueName\":\"my-q-items\",\"connection\":\"MyConnection\",\"type\":\"queueTrigger\",\"name\":\"qTrigger1\",\"direction\":\"in\"}")]
+        [InlineData("""{"queueName":"my-q-items","connection":"DefaultEndpointsProtocol=https;AccountName=a;AccountKey=b/c==;EndpointSuffix=core.windows.net","type":"queueTrigger","name":"queueTrigger1","direction":"in"}""", "{\"queueName\":\"my-q-items\",\"connection\":\"[Hidden Credential]\",\"type\":\"queueTrigger\",\"name\":\"queueTrigger1\",\"direction\":\"in\"}")]
+
```

Comment:
Could this be a breaking change in some scenarios? Let's say queue name is `"key=myqueue"` then we will replace it with `"[Hidden Credential]"`.

Reply:
We are sanitizing the entire binding object for all scenarios and not just the exception cases. If any extension introduces (or already has) a binding with name `"password="` or `"key="` (may not be good examples), they will be modified to `[Hidden Credential]` and the app can break.

In terms of binding values, I checked that the queue name and connection setting name cannot have "=" but they can have other characters like ":", so may be, we need to ensure that the Sanitizer class should never have tokens such as `"key:"`, otherwise we will modify that value to `[Hidden Credential]`. A unit test should work for this scenario.

Reply:
I am not sure I follow the last part of the comment. But for inputs like `"one:two;three"`, Sanitizer returns the same value output.

| Input              | Output             |
|-------------------|--------------------|
| one:two;three     | one:two;three      |
| one:a;two=three   | one:a;two=three    |
| foo=bar           | foo=bar            |

If you meant something else, can you share an example input?

Reply:
`key=myqueue` should return `[HiddenCredential]`. Let's also discuss this offline to make sure we don't have any confusion here.

Reply:
Right because `:` is the connection separator, so it won't be unexpected for connection names to include it. Now, if customer's have a section literally called "key" or "secret", or something else covered by our sanitizer that is another question.

Reply:
Maybe we should reconsider the approach taken here and sanitize only the binding values when we serve the response from the /functions/ endpoints.  Let's follow up offline.